### PR TITLE
Various Ansible Role fixes

### PR DESCRIPTION
**Pull request by redhatrises**
- Add CJIS profile
- Fix tags to be more accurate and comply with Galaxy v3
- Update old links and information

### CODE REVIEW

File: utils/ansible_galaxy_meta_template.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 galaxy_info:
-  author: SCAP Security Guide development team
+  author: Red Hat Security Automation development team
```
Comment (jan-cerny) [inline utils/ansible_galaxy_meta_template.yml]: ComplianceAsCode has a lot of contributors outside Red Hat.  I think we can't say that the author of the roles is only a Red Hat team. I prefer to put here "ComplianceAsCode contributors" or "ComplianceAsCode development team"

File: utils/ansible_galaxy_meta_template.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
   description: @DESCRIPTION@
 
   issue_tracker_url: https://github.com/OpenSCAP/scap-security-guide/issues
```
Comment (jan-cerny) [inline utils/ansible_galaxy_meta_template.yml]: When you edit this file, could you change this link as well?

File: utils/ansible_galaxy_meta_template.yml (hunk 1 above)
Comment (redhatrises) [reply utils/ansible_galaxy_meta_template.yml]: Good point! Fixed @jan-cerny

File: utils/ansible_galaxy_meta_template.yml (hunk 2 above)
Comment (redhatrises) [reply utils/ansible_galaxy_meta_template.yml]: Oops... missed that one. Fixed @jan-cerny

File: utils/ansible_galaxy_meta_template.yml [hunk 3]
```diff
@@ -20,10 +20,11 @@ galaxy_info:
...
 
   galaxy_tags:
+  - system
   - hardening
   - openscap
   - ssg
-  - scap-security-guide
+  - scap
   - security
   - compliance
```
Comment (shawndwells) [inline utils/ansible_galaxy_meta_template.yml]: + ComplianceAsCode?

Comment (redhatrises) [reply utils/ansible_galaxy_meta_template.yml]: @shawndwells fixed

**Comment by jan-cerny** [comment]
Thank you!

**Pull request merged**